cypress/support/server.js

```diff
@@ -148,25 +147,4 @@ export const mockGetS3Metadata = (items, shouldThrowError) => {
   ).as('getS3Metadata');
 };
 
-// intercept s3 file link and serve corresponding cypress fixture
-export const mockGetS3FileContent = (items, shouldThrowError) => {
-  cy.intercept(
-    {
-      method: DEFAULT_GET.method,
-      url: new RegExp(
-        `${parseStringToRegExp(S3_FILES_HOST, {
-          characters: ['.', '-'],
-        })}/[a-zA-Z0-1\\/]+\\.[a-z0-1]+$`,
-      ),
-    },
-    ({ reply, url }) => {
-      if (shouldThrowError) {
-        reply({ statusCode: StatusCodes.BAD_REQUEST });
-        return;
-      }
-
-      const filepath = url.slice(S3_FILES_HOST.length);
-      reply({ fixture: filepath });
-    },
-  ).as('getS3FileContent');
-};
+// bug: mockGetS3FileContent intercept static/js/bundle.js which fails tests
```

Comment:
What does this mean?

Reply:
I had an issue with the mocked endpoints. Even tough they are identical to compose's, the mocked endpoint providing s3-files was preventing react to load. We will need to fix this in the future, but currently removing the mocked endpoint does not impact the tests.